goplugin/plugin_name_builder.go

```diff
@@ -12,19 +12,25 @@ var pluginStorage storage
 // GetPluginFileNameToLoad check which file to load based on name, tyk version, os and architecture
 // but it also takes care of returning the name of the file that exists
 func GetPluginFileNameToLoad(path string, version string) string {
```

Comment:
@sredxny can we make this func returns an error if the file file doesn't exist in any of the paths?

Reply:
done